### ISSUE TITLE
Staging Review finding: Error is not specific

### ISSUE THREAD

**Issue by shiragoodman** [department-of-veterans-affairs/va.gov-team#127626]
_Need help? Please review [how to read a Staging Review ticket](https://depo-platform-documentation.scrollhelp.site/collaboration-cycle/anatomy-of-a-staging-review-issue-ticket). Tag `@platform-governance-team-members` on Slack if you need further assistance._

## Product Information
**Team:** BIO - Pingwind
**Product:** VA Form 21-8940 Veteran's Application for Increased Compensation Based on Unemployability
**Feature:** Form Digitization

## Findings details
**VA.gov Experience Standard - issue:** [User encounters content with errors or inconsistencies.](https://design.va.gov/about/experience-standards/#user-encounters-accurate-consistent-content)
**VA.gov Experience Standard - category:** [Consistency](https://design.va.gov/about/experience-standards/#category-consistency)
**Launch-blocking:** No
**Design System review:** No
**Collab Cycle Reviewer:** @jasonday (Accessibility), @kkitagawa-bosch (content)

### Description

Error messages should be explicit in the expectation and reference the required information.

The error for "Doctor's name(*Required)" on step 3 of 12 is "You must provide a response", which is generic and doesn't follow our error guidance. For example, leaving "Street address" blank, the error is "Enter a street address" which explicitly identifies how to correct the error.  

<details><summary>Link, screenshot or steps to recreate</summary>

<img width="302" height="188" alt="generic error" src="https://github.com/user-attachments/assets/9d8a3a11-3a19-404b-8bb9-8c2315168297" />

</details>

## Recommended action

Update the error message so that it provides clear suggestions for how to correct the error, and so that the user understands what the error is referencing. For example, "Enter a doctor's name"

## References

- **Accessibility Defect Severity:** [3: Medium. Should be fixed in 1-3 sprints post-launch.](https://depo-platform-documentation.scrollhelp.site/developer-docs/accessibility-defect-severity-rubric#Accessibilitydefectseverityrubric-a11y-defect-3)
- **WCAG Success Criteria:** [WCAG 3.3.3 Error Suggestion - Level AA](https://www.w3.org/TR/WCAG22/#error-suggestion)
- **Modality:** N/A
- **Design System Component:** N/A
- **Design System Pattern or Template:** N/A
- **Design System Foundation:** N/A
- **Content Style Guide:** N/A
- **Context:** N/A
- **VADS Guidance:** N/A

---

### Next Steps for the VFS Team

_How did this touchpoint go? [Give feedback on the Collaboration Cycle](https://ows.io/qs/o3jkwoez) at any time._
- [ ] **Questions?** For the most timely response, comment on Slack in your team channel tagging `@platform-governance-team-members` with any questions or to get help validating the issue.
- [ ] **Ticket ownership:** If you believe that this issue is out of scope, not your team's responsibility, or a Design System issue; or if you believe the finding should not be fixed; comment and tag `@platform-governance-team-members` on your team channel in Slack to provide an explanation and who you believe is responsible. The Governance team will follow up.
- [ ] **Close this ticket** when the issue has been resolved or validated by your Product Owner.
